README.md

```diff
@@ -105,6 +105,10 @@ To include your `avatar`, You just have to replace `<yourusername>` with your **
 
 For example: <http://github.com/eddiejaoude.png>
 
+### A simple gif version of adding your profile
+
+![Linkfree adding profile GIF](https://user-images.githubusercontent.com/51878265/165113896-35d34c1d-c95e-4bb4-b658-df1afb9ab658.gif)
```

Comment:
Should we capitalize the `f` here or keep it as-is?
```suggestion
![LinkFree adding profile GIF](https://user-images.githubusercontent.com/51878265/165113896-35d34c1d-c95e-4bb4-b658-df1afb9ab658.gif)
```

Reply:
Since this will be the alt text I don’t think it really matters.